ml-agents/mlagents/trainers/env_manager.py

```diff
@@ -78,7 +77,7 @@ def set_env_parameters(self, config: Dict = None) -> None:
 
     @property
     @abstractmethod
-    def external_brains(self) -> Dict[BehaviorName, BrainParameters]:
+    def external_behaviors(self) -> Dict[BehaviorName, BehaviorSpec]:
```

Comment:
I think external is no longer needed as it referred to external brain in a time when broadcasting was a thing. I would go for `behaviors` or `dict_behaviors`

Reply:
Should it be `trainable_behaviors` to differentiate it from inference or heuristic behaviors?

Reply:
inference and heuristic behaviors do not make it this far in the Python code...

Reply:
or `training_behaviors`? Just because it's training doesn't mean it's trainable :)

Reply:
`training_behaviors` sounds good. Guess I'd want to differentiate it from inference and heuristic so that it doesn't seem like calling this method will give you _all_ the behaviors in the scene, including the ones that don't make it to Python.